relay-sampling/src/lib.rs

```diff
@@ -1990,37 +2264,60 @@ mod tests {
     }
 
     #[test]
-    ///Test SamplingRule deserialization
-    fn test_nondecaying_sampling_rule_deserialization() {
+    fn test_non_decaying_sampling_rule_deserialization() {
+        let serialized_rule = r#"{
+            "condition":{
+                "op":"and",
+                "inner": [
+                    { "op" : "glob", "name": "releases", "value":["1.1.1", "1.1.2"]}
+                ]
+            },
+            "samplingValue": {"type": "sampleRate", "value": 0.7},
+            "type": "trace",
+            "id": 1
+        }"#;
+        let rule: Result<SamplingRule, _> = serde_json::from_str(serialized_rule);
+
+        assert!(rule.is_ok());
+        let rule = rule.unwrap();
```

Comment:
The test will fail either way:

```suggestion
        let rule = serde_json::from_str(serialized_rule).unwrap();
```